pkg/helpers/file.go

```diff
@@ -0,0 +1,22 @@
+package helpers
+
+import (
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+func WriteFile(path string, data []byte, perm fs.FileMode) error {
+	dir, _ := filepath.Split(path)
+	err := os.MkdirAll(dir, os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(path, data, perm)
```

Comment:
Since we're on Go 1.16, we can just use `os.WriteFile` and avoid importing `ioutil`: https://golang.org/src/io/ioutil/ioutil.go?s=1537:1605#L35

Reply:
Additionally, we may want to verify if the file exists already before writing, since `os.WriteFile` will simply truncate/zero an existing file before writing. Maybe error if file already exists? Can't decide if that's a bad idea or not.